griptape/drivers/vector/redis_vector_store_driver.py

```diff
@@ -61,7 +60,9 @@ def upsert_vector(
         key = self._generate_key(vector_id, namespace)
         bytes_vector = json.dumps(vector).encode("utf-8")
 
-        mapping = {"vector": np.array(vector, dtype=np.float32).tobytes(), "vec_string": bytes_vector}
+        mapping = {}
+        mapping["vector"] = np.array(vector, dtype=np.float32).tobytes()
+        mapping["vec_string"] = bytes_vector
```

Comment:
Why this change?

Reply:
@andrewfrench helped me out with this one, but it solves both `Argument of type "str" cannot be assigned to parameter "__value" of type "bytes" in function "__setitem__"` and `Argument of type "dict[str, bytes]" cannot be assigned to parameter "mapping" of type "Mapping[_Key, _Value]" in function "hset"`

Reply:
In all honesty this fix was more the result of trial and error than anything. The Pyright complaint was pretty resistant to any change except defining each key-value pair independently.